src/utils/KML.js

```diff
@@ -125,6 +125,14 @@ const sanitizeFeature = (feature) => {
         text.setTextAlign(feature.get('textAlign'));
       }
 
+      if (feature.get('offsetX')) {
```

Comment:
How about adding a `text` prefix?
```suggestion
      if (feature.get('textOffsetX')) {
```

Reply:
Applies also to offsetY.

Reply:
You think so? Since the ol attributes are only offsetX and offsetY and there are no other classes using these names, I would use the exact ol Attribute names

Reply:
Done